pages/index.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-	<article>
+	<div>
```

Comment:
because of this change. The article in `_layout.scss` associated with this element should be changed from article to div as well. 

As of now the white background for the article is not showing *see homepage*

Reply:
Resolved in a3b91a3.